app/controllers/admin/finance/statements_controller.rb

```diff
@@ -53,16 +53,16 @@ def statement_date
       filter_params[:statement_date].presence
     end
 
-    def output_fee
+    def fee_type
```

Comment:
This struck me as a little odd, if we want it to default to 'Output' we should just pre-set the filter I think.